### PR TITLE
Rename module networkadmin-service to network-admin-service

### LINKED ISSUE #6825: rename hedera-networkadmin-service to hedera-network-admin-service

### PR DESCRIPTION
**Pull request by kimbor**
Rename module hedera-networkadmin-service to hedera-network-admin-service
Rename module hedera-networkadmin-service-impl to hedera-network-admin-service-impl

**Related issue(s)**:

Fixes #6825 

**Notes for reviewer**:
This is a targeted PR to simply add a dash between network and admin in the modules previously known as networkadmin.

### CODE REVIEW

**Review by mhess-swl — APPROVED** [review]

**Review by povolev15 — APPROVED** [review]
LGTM

**Review by jeromy-cannon — APPROVED** [review]
from a release engineering perspective I approve the build changes, please wait for other approvers for remaining code before merging.

**Pull request merged**
